alert-policies/blazor-server/TransactionErrors.yml

```diff
@@ -0,0 +1,30 @@
+name: Transaction Errors
+
+description: |+
+  This alert is triggered when the the transactions fail more than 10% of the time in 5 minutes.
```

Comment:
"This alert is triggered when the transactions fail more than 10% of the time in 5 minutes."